reactos/drivers/usb/usbport/usbport.c

```diff
@@ -2275,21 +2283,11 @@ USBPORT_MapTransfer(IN PDEVICE_OBJECT FdoDevice,
     Transfer->SgList.CurrentVa = CurrentVa;
 
     Mdl->MdlFlags |= MDL_MAPPING_CAN_FAIL;
-
-    if (Mdl->MdlFlags & (MDL_SOURCE_IS_NONPAGED_POOL |
-                         MDL_MAPPED_TO_SYSTEM_VA))
-    {
-        MappedSystemVa = Mdl->MappedSystemVa;
-    }
-    else
-    {
-        MappedSystemVa = MmMapLockedPages(Mdl, KernelMode);
-    }
+    MappedSystemVa = MmGetSystemAddressForMdl(Mdl);
```

Comment:
The MmGetSystemAddressForMdl routine is obsolete for Windows 2000 and later versions of Windows, and for Windows Me. It is supported only for WDM drivers that must run on Windows 98. Otherwise, use MmGetSystemAddressForMdlSafe.